documentation/schema.yaml

```diff
@@ -70,6 +70,9 @@ schema:
         $ref: '#/components/schemas/AssociatedArtifact'
     rightsHolder:
       type: string
+    duplicateOf:
+      type: string
+      format: uri
```

Comment:
I guess duplicationOf and rightsHolder.duplicateOf will not cause problems with the taught of jsonLd context?